components/homme/src/theta-l/imex_mod.F90

```diff
@@ -246,15 +246,15 @@ subroutine compute_stage_value_dirk(n0,np1,alphadt,qn0,dt2,elem,hvcoord,hybrid,&
                 do k=1,nlev-1
                    dphi(i,j,k)=dphi(i,j,k) + x(k+1,i,j)-x(k,i,j)
                 enddo
-                dphi(i,j,nlev)=dphi(i,j,nlev) + (0 - x(k,i,j) )
```

Comment:
why did this work before? was k updated to nlev anyway?

Reply:
Yes, k was updated to nlev. I imagine that happens, in practice, with all compilers, but I don't think it's in the standard. (I might be wrong.)